vllm/utils.py

```diff
@@ -109,8 +111,9 @@ def touch(self, key: Hashable) -> None:
     def get(self,
             key: Hashable,
             default_value: Optional[T] = None) -> Optional[T]:
+        value: Optional[T]
```

Comment:
this doesn't seem to be optional anymore this way?

Reply:
(iiuc if there's no item, it should raise an exception?)

Reply:
Yes, that is the intention so it works more like a Python dictionary.